packages/block-library/src/navigation-link/edit.js

```diff
@@ -193,6 +222,14 @@ function NavigationLinkEdit( {
 					/>
 				</PanelBody>
 			</InspectorControls>
+			<BlockNavigationListItemFill>
+				<BlockNavigationListItem
+					wrapperComponent="div"
+					onClick={ noop }
+				>
+					{ editField }
```

Comment:
It's cool that this works so well.

The only thing I noticed is that being able to format the text (using shortcuts) is a bit unusual in the navigator, though I'm not sure how that could be handled in any other way.